.circleci/config.yml

```diff
@@ -342,9 +316,6 @@ jobs:
       <<: *ENVIRONMENT
     steps:
       - checkout
-      - add_ssh_keys:
-          fingerprints:
-            - "76:94:15:c8:a0:41:f0:8b:9b:f1:f7:36:4c:d5:7f:57"
```

Comment:
It looks like this is actually required as `publish-static-assets` wants to push a commit back into the repo: https://app.circleci.com/pipelines/github/hashicorp/waypoint/6439/workflows/a20dc52f-c62a-44aa-ab5d-5c47e4b20059/jobs/50414